src/RA_AchievementOverlay.cpp

```diff
@@ -1281,10 +1281,10 @@ void AchievementOverlay::Render(HDC hRealDC, RECT* rcDest) const
 
         //	Draw control text:
         sprintf_s(buffer, 1024, " ->:%s ", "Next");
-        TextOut(hDC, nRightPx - nControlsX1, nControlsY1, NativeStr(buffer).c_str(), strlen(buffer));
+        TextOut(hDC, nRightPx - nControlsX1, nControlsY1, NativeStr(std::string{ buffer }).c_str(), strlen(buffer));
 
         sprintf_s(buffer, 1024, " <-:%s ", "Prev");
-        TextOut(hDC, nRightPx - nControlsX1, nControlsY2, NativeStr(buffer).c_str(), strlen(buffer));
+        TextOut(hDC, nRightPx - nControlsX1, nControlsY2, NativeStr(std::string{ buffer }).c_str(), strlen(buffer));
```

Comment:
Why wrap buffer in a `std::string`? `NativeStr` accepts a `NativeCString`, which if I understand it is just a `const char*` (i.e. buffer). Converting to a `std::string` seems like overhead that can be avoided.

Reply:
OK I must have misunderstood the concept, for some reason I thought the argument inside NativeStr was supposed be std::string but then realized I made overloads for c strings.